lib/spack/spack/spec.py

```diff
@@ -973,7 +973,7 @@ def _libs_default_handler(descriptor, spec, cls):
     name = spec.name.replace('-', '?')
 
     # Avoid double 'lib' for packages whose names already start with lib
-    if not name.startswith('lib'):
+    if not name.startswith('lib') and not is_windows:
```

Comment:
This should be `and if not self.satisfies('platform=windows')` (i.e. use the spec info rather than checking the actual system platform when possible).

Reply:
So when I try that I get an error saying self isn't defined in that file. I also see is_windows defined earlier in the file, which is why I thought this syntax was correct